openapi/adintel/adintel.yaml

```diff
@@ -0,0 +1,267 @@
+openapi: 3.0.0
+info:
+  title: Advertising Intelligence REST APIs
+  version: '1.0'
+  description: ''
+servers:
+  - url: 'https://prod.apigateway.co/products/adintel'
+    description: Production
+  - url: 'https://demo.apigateway.co/products/adintel'
+    description: Demo
+  - url: '{local}/products/adintel'
+    description: Local
+  - url: 'http://localhost:11001/products/adintel'
+    description: Localhost
+paths:
+  /accountStats:
+    parameters: []
+    get:
+      summary: Get Account Stats
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                description: ''
+                type: object
+                x-examples:
+                  example-1:
+                    data: []
+                    links:
+                      self: ''
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/accountStats'
+                  links:
+                    type: object
+                    required:
+                      - self
+                    properties:
+                      self:
+                        type: string
+                required:
+                  - data
+                  - links
+              examples: {}
+      operationId: get-accountStats
+      description: Retrieve account stats for all connected advertising accounts under a specific provider
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Content-Type'
+        - $ref: '#/components/parameters/startAt'
+        - $ref: '#/components/parameters/endAt'
+        - $ref: '#/components/parameters/businessLocation.id'
+        - $ref: '#/components/parameters/provider'
+        - $ref: '#/components/parameters/connectedAccount.resourceId'
+        - $ref: '#/components/parameters/pageLimit'
+        - $ref: '#/components/parameters/pageCursor'
+      security:
+        - OAuth2:
+            - adintel
+  '/accountStats/{connectedAccount.id}':
+    parameters:
+      - $ref: '#/components/parameters/connectedAccount.id'
+    get:
+      summary: Get Account Stats for one Connected Account
+      tags: []
```

Comment:
We have been tagging the operations and models with the model name. That will group them together in the generated documentation.